wordpress-mobile/WordPress-iOS
# Stock Photos: Tracks events

Implements part of #8954 , Tracks events

I am not sure, in particular, of the way I am tracking the upload event. 

To test:
- Access Stock Photos, from the Media Library and Aztec editor. Confirm an event is tracked.
- Pick a photo from Stock Photos, it will be uploaded to the Media Library. Confirm an event is tracked.
- Insert a Stock Photo directly into the editor. An event should be tracked as well.
